onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -53,6 +53,7 @@
 import org.onebusaway.android.map.RouteMapController;
 import org.onebusaway.android.map.StopMapController;
 import org.onebusaway.android.region.ObaRegionsTask;
+import org.onebusaway.android.ui.HomeActivity;
```

Comment:
@azizmb9494 Could you please make sure you have "Optimize imports" checked in Android Studio when you go to commit the changes?  It should automatically remove any unused imports like this when you commit.